js/contraster.js

```diff
@@ -21,6 +22,12 @@ bcpl.contraster = (($, browserStorage) => {
 	 */
 	const contrastButtonClickHandler = (clickEvent) => {
 		const settings = clickEvent ? clickEvent.data : contrasterDefaults;
+		const $eventTarget = $(clickEvent.currentTarget);
+
+		if ($eventTarget.is(contrasterDefaults.selectors.toggleText)) {
+			$eventTarget.parent().find('input').trigger('click');
```

Comment:
i mean is it possible to use closest here?

Reply:
Technically yes, but I just need the immediate parent.